bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/ThingProperty.java

```diff
@@ -0,0 +1,28 @@
+package org.eclipse.smarthome.core.thing;
+
+import org.eclipse.smarthome.config.core.Configuration;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandlerFactory;
+
+/**
+ * The {@link ThingProperty} enumeration defines all base configuration properties for a {@link Thing}. They
+ * will be injected into the {@link Configuration} of a thing during its instantiation (compare
```

Comment:
I would remove lines 2-4 of this JavaDoc. The enum only defines some default keys for properties - nothing more and nothing less.